contracts/vault/IVault.sol

```diff
@@ -89,6 +89,11 @@ interface IVault {
      */
     function totalPrincipal() external view returns (uint256);
 
+    /**
+     * The accumulated performance fee amount.
+     */
+    function accumulatedPerfFee() external view returns (uint256);
+
```

Comment:
you forgot to add the `override(IVault)` modifier to the accumulatedPerfFee variable in the Vault.sol file